### ISSUE TITLE
Remove color literals from Sass code (convert to variables)

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#289]
Ref: https://github.com/mozilla/idea-town/pull/275

We have a couple hard coded values in the Sass which will need to be extracted into variables, plus it seems like Sass Lint doesn't like using `rgba($black, .4)` in our Sass, and wants those extracted into variables as well.

Current status:

```
idea_town/frontend/static-src/styles/modules/_buttons.scss
  38:14  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals

✖ 1 problem (0 errors, 1 warning)


idea_town/frontend/static-src/styles/modules/_cta-view.scss
  26:14  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals

✖ 1 problem (0 errors, 1 warning)


idea_town/frontend/static-src/styles/modules/_modals.scss
   34:28  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
   71:15  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
   81:15  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
  134:16  warning  Color literals such as 'bbb' should only be used in variable declarations    no-color-literals
  141:27  warning  Color literals such as 'bbb' should only be used in variable declarations    no-color-literals

✖ 5 problems (0 errors, 5 warnings)


idea_town/frontend/static-src/styles/modules/_settings.scss
   3:15  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
   4:21  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
  16:17  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals
  52:31  warning  Color functions such as 'rgba' should only be used in variable declarations  no-color-literals

✖ 4 problems (0 errors, 4 warnings)
```

**Comment by johngruen** [comment]
@meandavejustice why good first bug and good-first-bug?

**Comment by meandavejustice** [comment]
@johngruen They were already both added, I wanted people to see all bugs while filtering with both, before I went through and cleaned up and removed the extra label.

**Comment by johngruen** [comment]
Closed via #472
